src/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -2257,6 +2259,11 @@ private void triggerError(String message) {
             callback.accept(message);
     }
 
+    /** Sets where to locate external instance data: A path to be placed in front of what follows `jr://file/` in the src attribute of the instance element */
```

Comment:
It's not clear how to use this method. Should users provide a String with or without the `jr://file` prefix? Maybe add an example to the doc block?

Reply:
Good suggestion. Thanks.